Libraries/CO.CDP.GovUKNotify/GovUKNotifyApiClient.cs

```diff
@@ -26,7 +32,16 @@ public GovUKNotifyApiClient(IHttpClientFactory httpClientFactory, IAuthenticatio
     public async Task<EmailNotificationResponse?> SendEmail(EmailNotificationRequest request)
     {
         var res = await client.PostAsJsonAsync("/v2/notifications/email", request, jsonSerializerOptions);
-        res.EnsureSuccessStatusCode();
-        return await res.Content.ReadFromJsonAsync<EmailNotificationResponse>();
+        try
+        {
+            res.EnsureSuccessStatusCode();
+            return await res.Content.ReadFromJsonAsync<EmailNotificationResponse>();
+        }
+        catch (Exception ex)
+        {
+            var content = await res.Content.ReadAsStringAsync();
+            _logger.LogError($"GovUKNotify error, response status: {res.StatusCode}, response body: {content}, exception: {ex}");
```

Comment:
Since we're using Serilog and send JSON formatted logs to the CloudWatch, so you can provide extra context as an array or serializable object, instead of a string :+1: